includes/header.php

```diff
@@ -59,41 +59,45 @@
     <script src="/assets/js/highlight.pack.js"></script>
     <!-- Page-specific CSS and JS -->
     <?php if(isset($import_moment) && $import_moment): ?>
-    <script src="/assets/js/moment.js"></script>
-    <script src="/assets/js/moment-timezone-with-data-10-year-range.js"></script>
+      <script src="/assets/js/moment.js"></script>
+      <script src="/assets/js/moment-timezone-with-data-10-year-range.js"></script>
     <?php endif;
-    if(isset($import_leaflet) && $import_leaflet): ?>
-    <link href="/assets/css/leaflet.css" rel="stylesheet">
-    <link href="/assets/css/leaflet.fullscreen.css" rel="stylesheet">
-    <script src="/assets/js/leaflet.js"></script>
-    <script src="/assets/js/Leaflet.fullscreen.min.js"></script>
+    <?php if(isset($import_leaflet) && $import_leaflet): ?>
```

Comment:
```suggestion
    if(isset($import_leaflet) && $import_leaflet): ?>
```
the only error I could find.